packages/backend-test-utils/src/database/TestDatabases.ts

```diff
@@ -44,6 +44,7 @@ const LARGER_POOL_CONFIG = {
 export class TestDatabases {
   private readonly instanceById: Map<string, Instance>;
   private readonly supportedIds: TestDatabaseId[];
+  static defaultIds?: TestDatabaseId[];
```

Comment:
```suggestion
  private static defaultIds?: TestDatabaseId[];
```

this gets it out of the api report too

Reply:
Cool, thanks 👍